### ISSUE TITLE
some problems about best.pt and last.pt

### ISSUE THREAD

**Issue by xiaoxuebajie** [ultralytics/yolov5#1004]
## ❔Question
last.pt and best.pt have the same epoch='-1' and the same best_fitness

## Additional context
![Screenshot from 2020-09-21 18-23-31](https://user-images.githubusercontent.com/26498776/93756508-cc58e480-fc37-11ea-97e5-769f7c3d1973.png)

**Comment by xiaoxuebajie** [comment]
In my expriment, the epochs in best and last are not same

**Comment by glenn-jocher** [comment]
@xiaoxuebajie checkpointing operates correctly. 

There is no constraint that best and last must be different.

**Comment by xiaoxuebajie** [comment]
@glenn-jocher I mean the epoch in the best.pt and last.pt are always '-1'.

**Comment by glenn-jocher** [comment]
@xiaoxuebajie yes, this is a final step performed after stripping the optimizer from the checkpoint at the end of a successful training.
https://github.com/ultralytics/yolov5/blob/b8f656742db6dd9f22ece7461014b43bfdcf386a/utils/general.py#L673-L685


This allows this model to be used pretrained weights for further trainings, the same as the official COCO-trained models we make available.

**Comment by tailtq** [comment]
@glenn-jocher Hi, thanks for your great work. Are there any ways to "undo" this process? In my case, I want to continue training my stripped last.pt because I think it still has the potential to converge more. Thanks

**Comment by glenn-jocher** [comment]
@tailtq no this can not be undone. The cosine LR scheduler is planned for the exact number of epochs you specify, you can not continue it once complete as the LR is basically at zero.

**Comment by tailtq** [comment]
@glenn-jocher Thanks, I continued my training with the previous experiment.
